TaxiFare.ipynb

```diff
@@ -64,6 +71,7 @@
     "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json\" \r\n",
     "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json\" \r\n",
     "\r\n",
+    "// Install the necessary NuGet packages for training ML.NET model and plotting\r\n",
     "#r \"nuget:Microsoft.ML,1.5.1\"\r\n",
```

Comment:
Put a markdown cell above the package cell with the heading 
Install necessary NuGet packages